pkg/machine/applehv/vfkit.go

```diff
@@ -38,7 +40,18 @@ func getDefaultDevices(mc *vmconfigs.MachineConfig) ([]vfConfig.VirtioDevice, *d
 	if err != nil {
 		return nil, nil, err
 	}
-	devices = append(devices, disk, rng, serial, readyDevice)
+
+	if runtime.GOARCH == "arm64" {
+		rosetta := &vfConfig.RosettaShare{
+			DirectorySharingConfig: vfConfig.DirectorySharingConfig{
+				MountTag: define.MountTag,
+			},
+			InstallRosetta: true,
+		}
+		devices = append(devices, disk, rng, serial, readyDevice, rosetta)
+	} else {
+		devices = append(devices, disk, rng, serial, readyDevice)
+	}
```

Comment:
This is duplicating a bunch of code for no reason
This really should be written as
```
devices = append(devices, disk, rng, serial, readyDevice)
if arm64 {...,devices = append(devices, rosetta)}
```

Reply:
fixed